alpine-php/k8s/Dockerfile.tmpl

```diff
@@ -0,0 +1,163 @@
+FROM unocha/alpine-base-nginx-extras:%%UPSTREAM%%
```

Comment:
Unless you require nginx > 1.14.2, you could instead use `unocha/nginx:1.14.2`.
You get a slightly newer upload progress module (0.9.2 instead of 0.9.1) and the nginx embedded lua.
My initial intention when adding `unocha/nginx` was to get rid of the other nginx images. :)

Reply:
You could alternatively start from a suitable `unocha/php7` image and just add the following packages: `nginx`, `nginx-mod-http-upload-progress` and `nginx-mod-http-lua`

Reply:
Ah, I had a look at the existing nginx containers, and their metadata suggests they're Alpine 3.6 :-/

Reply:
metadata must be wrong, 1.14.2 is on 3.8 at least